index.html

```diff
@@ -0,0 +1,717 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/668d372fa7.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" type="text/css" href="main.css" media="screen and (min-width: 1025px)">
+    <link rel="stylesheet" type="text/css" href="tab.css" media="screen and (max-width: 1024px) and (min-width: 766px)">
+    <link rel="stylesheet" type="text/css" href="phone.css" media="screen and (max-width: 765px)">
+    <title>The Next Web Page Clone</title>
+</head>
+
+<body>
+    <div class="fixed">
+        <header class="hd">
+            <a href="#" class="img-wrapper">
+                <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg " alt="TNW">
+            </a>
+            <div>
+                <div class="hdTop">
+
+                    <div id="hdLine1">
+
+                        <ul class="hdLine1-left">
+
+                            <li class="blank"><img
+                                    src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg "
+                                    alt="TNW"> </li>
+                            <li><a href="#">News</a></li>
+                            <li><a href="#">TNW2020</a></li>
+                            <li class="business_li"><a href="#">Business ▾</a>
+                                <ul class="business_opt">
+                                    <li><a href="#">Index</a></li>
+                                    <li><a href="#">TNW X</a></li>
+                                </ul>
+                            </li>
+                            <li><a href="#">AMAs</a></li>
+                            <li><a href="#">Space</a></li>
+                            <li><a href="#">Terms & Consitions</a></li>
+                        </ul>
+
+                        <ul class="hdLine1-left1">
```

Comment:
[Required] please use semantic tags instead of just div tags. [reference](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/semantic_html_requirement.png)

[TIPS] **nav** tag for navigations and **main** tag for the main content of the page.